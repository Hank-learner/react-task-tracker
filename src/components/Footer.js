import { Link } from 'react-router-dom'

const Footer = () => {
    return (
        <footer>
            <p>Developed with help of Traversy Media &copy; 2021</p>
            <Link to='/about'>About</Link>
        </footer>
    )
}

export default Footer
