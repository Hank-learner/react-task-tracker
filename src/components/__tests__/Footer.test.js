import renderer from 'react-test-renderer';
import Footer from '../Footer';

// const validProps = {

// }


describe('Footer snapshot test', () => {
    it('renders correctly', () => {
        const tree = renderer.create(Footer).toJSON();
        expect(tree).toMatchSnapshot();
    });
});