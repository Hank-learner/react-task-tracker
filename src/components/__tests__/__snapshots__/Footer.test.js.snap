// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot test renders correctly 1`] = `null`;
